packages/core/admin/server/middlewares/rateLimit.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const utils = require('@strapi/utils');
+
+const { RateLimitError } = utils.errors;
+
+module.exports =
+  (config, { strapi }) =>
+  async (ctx, next) => {
+    const ratelimit = require('koa2-ratelimit').RateLimit;
+
+    return ratelimit.middleware({
+      interval: { min: 15 },
+      max: 5,
```

Comment:
So it would be max 5 password attempts per 15 minutes? Is it too strict?

Reply:
That's the default yes but it is user configurable via `./config/admin.js`

Reply:
The default seems to be 5 per minute no? https://github.com/ysocorp/koa2-ratelimit#configuration
I would say it can happen to legitimately try more than 5 times you password within 15 min. What about 5 minutes instead?

Reply:
That should be fine, I was trying to be a bit more cautious but since it's configurable then users can make it more secure.